object/types.proto

```diff
@@ -160,3 +160,21 @@ message Object {
   // Payload bytes.
   bytes payload = 4 [json_name = "payload"];
 }
+
+// Meta information of split hierarchy for object assembly. With right child
```

Comment:
Suggest to use "last object in the split list or something similar.

Reply:
Fixed.